[BUG?] 不能直接使用go get

## 错误信息
```
main.go:4:2: import "github.com/sumwai/wox" is a program, not an importable package
```

## 源代码
```go
package main

import (
	wox "github.com/sumwai/wox"
)

func main() {
	wox.NewPM().Load("plugins").Run()
}
```

@sumwai 请解决该问题。

--- a/main.go
+++ b/main.go
@@ -1,5 +1,0 @@
-package main
-
-func main() {
-	NewPM().Load("plugins").Run()
-}

--- a/plugin.go
+++ b/plugin.go
@@ -1,4 +1,4 @@
-package main
+package wox
 
 import (
 	"errors"

